README.md

```diff
@@ -66,17 +110,47 @@ Required environment variables:
  - AWS_SECRET_ACCESS_KEY
 
 
+### Mobros command: install <a id="cmd-install"/>
+
+`Requires escalated privilages`
+
+Example of usage:
+```
+mobros install --pkg_list package_1=0.0.0-0 package_2 package_3=1.1.0-4
+```
+
+mobros install is used to install complex dependency trees of debian packages. Before installing packages, mobros scans the dependency of the requested packages, calculates candidates for installation and in the case of version conflicts, it produces reports for the user to make decisions.
+
+Generated artifacts:
+- Generates a file called "tree.mobtree" where the command was executed, with a resume'd dependency tree of what the packages the user requested.
+
+#### Conflict Reporting <a id="cmd-install-conflict-report"/>
+
+![image](https://user-images.githubusercontent.com/84720623/231483118-44587cbf-3e3f-46fe-9f9c-c1a5329ed1a9.png)
+
+Mobros analyses the dependency tree, and takes in consideration the packages that are installed. The purpose is to have a cautious upgrade of the environment packages, reducing possible impact to a minimum.
+To further troubleshoot where this clash is coming from, its sugested to analyze the tree.mobtree file.
+
+#### Conflict Handling <a id="cmd-install-conflict-handling"/>
+
+Now that you are facing a conflict in the dependency tree, you need to take a decision and provided it to mobros.
+To understand how to solve you must first understand what mobros expects from the inputs he receives:
+- Package with a version (package_name=package_version): Mobros assumes that the user really wants this specific version, upgrading/downgrading it if it's installed.
```

Comment:
maybe be add the fact that the `package_version` needs to be fully correct:
- including build_id
- no wildcard support
- matching the repository name